packages/in_app_purchase/lib/src/store_kit_wrappers/sk_payment_queue_wrapper.dart

```diff
@@ -41,6 +41,12 @@ class SKPaymentQueueWrapper {
     callbackChannel.setMethodCallHandler(_handleObserverCallbacks);
   }
 
+  /// Get pending transactions.
```

Comment:
nit: Can this be explained in any more detail? Even if it's just linking to https://developer.apple.com/documentation/storekit/skpaymentqueue/1506026-transactions?language=objc.

Reply:
done